### PR TITLE
[kube-prometheus-stack] add scheme and tlsConfig to node exporter ServiceMonitor

### PR DESCRIPTION
**Pull request by rcanderson23**
#### What this PR does / why we need it:
This adds the ability to configure TLS and scheme to the node exporter ServiceMonitor.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 19.2.3
+version: 19.2.4
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:21]: new feature should raise minor version

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/Chart.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 19.2.3
+version: 19.3.3
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:21]:
```suggestion
version: 19.4.0
```

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
please fix dco: https://github.com/prometheus-community/helm-charts/pull/1488/checks?check_run_id=4164371258

**Comment by rcanderson23** [comment]
Followed the instructions in the link but doesn't appear to have signed off the commit where it is missing. Any suggestions on how to resolve this?

**Comment by monotek** [comment]
Rebase to main, squash and and force push....

**Comment by mrueg** [comment]
kube-prometheus-stack is using now servicemonitor of the node-exporter chart. Do you want to target this one instead?

**Pull request closed without merging**